cli/sawtooth_cli/version.py

```diff
@@ -0,0 +1 @@
+VERSION = """0.8.6.dev1"""
```

Comment:
The reason for switching away from "pkg_resources.get_distribution('sawtooth-cli').version" is that I want to have an explicit version file. But, yes, you are right. This makes the problem.  I'll switch back.